modules/effects/src/provide_effects.ts

```diff
@@ -41,35 +46,33 @@ import { rootEffectsInit as effectsInit } from './effects_actions';
 export function provideEffects(
   ...effects: Type<unknown>[]
 ): EnvironmentProviders {
-  return {
-    ɵproviders: [
-      effects,
-      {
-        provide: ENVIRONMENT_INITIALIZER,
-        multi: true,
-        useValue: () => {
-          inject(ROOT_STORE_PROVIDER);
-          inject(FEATURE_STATE_PROVIDER, { optional: true });
+  return makeEnvironmentProviders([
```

Comment:
Angular v15 now exports the `EnvironmentProviders` type, also there is the `makeEnvironmentProviders` function that converts providers array to environment providers.